src/routes/alerts.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const Alerts = require('../models/Alerts')
+const router = express.Router();
+
+router.get('/alerts', (req, res) => {
+    let alerts = new Alerts();
+
+    alerts.getAlerts()
+    .then(alerts => {
+      console.log(alerts);
```

Comment:
@Sanderson239 please remove this line, thanks! Also, because this line made it in it leads me to remind you to please run your linter before committing, thanks!